typescript-react/src/types/global.d.ts

```diff
@@ -11,7 +11,7 @@ declare interface INumberFormatter {
 // Custom branded types
 declare type DateStr = string & { __DateStrBrand__: void };
 declare type TimestampStr = string & { __TimestampStrBrand__: void };
-declare type Long = number & { __LongBrand__: void };
+declare type Long = string & { __LongBrand__: void };
```

Comment:
Are we aware of consequences from this?
Technically decimal can also be larger than JS allows and it should be treated similarly to this

Reply:
Decimals are also strings
```
declare type DecimalStr = string & { __DecimalStrBrand__: void };
```
Maybe we could rename `Long` to `LongStr` in the future to be more consistent, I didn't do it now since it would break existing type usage.